pkg/repo/migrations/helpers.go

```diff
@@ -1,35 +1,33 @@
 package migrations
 
 import (
-	"fmt"
+	"errors"
 	"os"
 	"path/filepath"
 
+	"github.com/libp2p/go-libp2p/core/peer"
+
 	"github.com/bacalhau-project/bacalhau/pkg/config"
 	"github.com/bacalhau-project/bacalhau/pkg/config/types"
 	"github.com/bacalhau-project/bacalhau/pkg/repo"
-	"github.com/libp2p/go-libp2p/core/peer"
-	"github.com/spf13/viper"
 )
 
-func getViper(r repo.FsRepo) (*viper.Viper, error) {
+func getConfigContext(r repo.FsRepo) (config.Context, error) {
 	repoPath, err := r.Path()
 	if err != nil {
 		return nil, err
 	}
 
-	configFile := filepath.Join(repoPath, config.ConfigFileName)
-	v := viper.New()
-	v.SetTypeByDefaultValue(true)
-	v.SetConfigFile(configFile)
-
+	configFile := filepath.Join(repoPath, config.FileName)
+	c := config.New()
```

Comment:
this will load config with default values and env variables. The intention here is to just get the configs from the config file